src/samplers/ratelimiting_sampler.js

```diff
@@ -32,7 +32,7 @@ export default class RateLimitingSampler {
         }
 
         this._maxTracesPerSecond = maxTracesPerSecond;
-        this._rateLimiter = new RateLimiter(maxTracesPerSecond);
+        this._rateLimiter = new RateLimiter(maxTracesPerSecond, 2.0);
```

Comment:
why `2.0`? If the `RateLimitingSampler` itself does not take this as argument, then `maxBalance` should probably be somehow calculated from `maxTracesPerSecond`